plugins/modules/sns_topic.py

```diff
@@ -287,9 +295,17 @@ def _topic_arn_lookup(self):
                 return topic
 
     def _create_topic(self):
+        attributes = { 'FifoTopic': 'false' }
+        tags = []
```

Comment:
just to showcase that the create method can also take some tags. Feature could be added later